en/datatypes/bitset.adoc

```diff
@@ -94,62 +97,53 @@ charset: func ["Shortcut for `make bitset!`"
 ]
 ```
 
-`Charset` is a shortcut for `make bitset!`, with the exception that `binary!` values are not allowed.
++
 
+`Charset` is a shortcut for `make bitset!`, with the exception that `binary!` values are not allowed.
++
 ```red
 >> charset #{2A}
 *** Script Error: charset does not allow binary! for its spec argument
 *** Where: charset
 *** Stack: charset  
 ```
-
++
 ```red
 >> charset [120 "hello" #"A"]   ; create and set bits using different value representations
 == make bitset! #{00000000000000004000000004890080}
 ```
-
++
 ```red
 >> charset [#"0" - #"9" #"a" - #"z"]   ; create a bitset using ranges of values
 == make bitset! #{000000000000FFC0000000007FFFFFE0}  
 ```
 
 == Complements
 
-In order to cope with the wide range of Unicode characters, bits outside of the bitsets are treated as virtual bits
-so they can be tested and set without errors. The bitset size will auto-expand according to the needs. 
-
-This is still not enough to deal with big ranges such as a bitset for `all Unicode characters except digits`. For such cases, it is possible to define a complemented bitset that represents the complement range of the specified bits. This makes it possible to have large bitsets while using only a tiny memory portion.
-
-
 Complemented bitsets can be created by including the word `not` in a block spec.
 
 ```red
 charset [not "0123456789"]   ; all characters except digits
 == make bitset! [not #{000000000000FFC0}]
 ```
 
-`Complement` can also be used to create complemented bitsets.
+The `action!` value `complement` can also be used to create complemented bitsets.
```

Comment:
Let's think about "The action value X" wording. "The complement action" is more direct, but what is wrong with just `Complement`, as it was originally here?